e2e-polybft/bridge_test.go

```diff
@@ -96,10 +77,10 @@ func executeStateSync(t *testing.T, client *jsonrpc.Client, txRelayer txrelayer.
 
 	t.Log("State sync proofs:", stateSyncProof)
 
-	input, err := types.ExecuteBundleABIMethod.Encode([2]interface{}{stateSyncProof.Proof, stateSyncEventsToAbiSlice(stateSyncProof.StateSync)})
+	input, err := types.ExecuteStateSyncABIMethod.Encode([2]interface{}{stateSyncProof.Proof, stateSyncProof.StateSync.ToMap()})
```

Comment:
You can use `stateSyncProof.EncodeAbi()` here.

Reply:
You sure?

Reply:
You're right done in https://github.com/0xPolygon/polygon-edge/pull/1013/commits/4d99c07b504640117cc9121f63c3b0e065a3e45b